plugins/git/webgui/js/gitMenu.js

```diff
@@ -30,8 +31,37 @@ function (topic, Menu, MenuItem, PopupMenuItem, model, viewHandler) {
     }
   };
 
-  viewHandler.registerModule(team, {
+  viewHandler.registerModule(nodeMenu, {
     type     : viewHandler.moduleType.TextContextMenu,
     priority : 70
   });
+
+  var fileMenu = {
+    id : 'git-text-team-file',
+    render : function (fileInfo) {
+      if (fileInfo.type === "Dir")
+        return;
+
+      var submenu = new Menu();
+
+      submenu.addChild(new MenuItem({
+        label : 'Blame',
+        onClick : function () {
+          topic.publish('codecompass/gitblame', {
+            fileInfo : fileInfo
+          });
+        }
+      }));
+
+      return new PopupMenuItem({
+        label : 'Team',
```

Comment:
Also, @bruntib, why do we call this "Team"?

Reply:
Yes, if we want then we can rename it to something else which describes better that this is a Git feature.

Reply:
I think Team is fine, but if we look for another name, simply "Git" could be appropriate.